src/components/all-legends/all-legends.tsx

```diff
@@ -76,11 +76,11 @@ export const AllLegends: React.FC = () => {
 
     // console.log(legendSale);
 
-    // const legendOffer = await marketplaceContract.makeLegendOffer(
-    //   nftContract,
-    //   legendId,
-    //   { value: price }
-    // );
+    const legendOffer = await marketplaceContract.makeLegendOffer(
```

Comment:
you can just run the function without instantiating it with const if you don't need ```legendOffer``` for anything. but maybe we should return legendOffer incase an error comes back?

Reply:
The write calls return an object with some data (hash, blocknumber, etc) which we may or may not want to use in the future. So either way, we can always change it back if we choose to use the returned object 